client/landing/gutenboarding/index.tsx

```diff
@@ -278,8 +276,8 @@ async function setupTranslationChunks( localeSlug: string, translatedChunks: str
 
 		return getTranslationChunkFile( chunkId, localeSlug, window.BUILD_TARGET ).then(
 			( translations ) => {
-				setLocaleData( translations );
```

Comment:
`react-i18n` doesn't replace this entirely because it's only available in react contexts. There are places where we use non-reactive translations with no concepts of "rerendering."

Anywhere that the `react-i18n` translations are updated, `setLocaleData` should also happen.

Reply:
ahhhh ok, thanks I was confused about that!